src/data/links.yaml

```diff
@@ -1443,6 +1443,40 @@ links:
       transacted and advertised in the retail market, protecting both the buyer
       and the seller.
 
+  - title: TenTree
+    url: https://www.tentree.ca/
+    featured: true
+    categories:
+      - clothing
+    tags: [clothing]
+    description: >-
+      Ten tree is a brand that actively promotes environmental stewardship. They have a wide variety of comfortable clothes and accessories and with each purchase, they pledge to plant 10 trees.
+
+  - title: Arvin  Goods
+    url: https://www.arvingoods.com/
+    featured: true
+    categories: 
+      - clothing
+    tags: [clothing]
+    description: >-
+      Brand that makes fashionable socks through a sustainable supply chain process they have developed. They have removed excess water from their manufacturing process - leaving more water left for communities to drink!
+
+  - title: Siizu
+    url: https://siizu.com/
+    categories:
+      - clothing
+    tags: [clothing]
+    description: >-
+      A mininalistic-styled brand that creates clothing with organic and natural fabrics. They use an ethical manufacturing process by having their factories close to each other and their shipping only uses recyclable packaging.
+
+  - title: For Days
+    url: https://www.fordays.com/
```

Comment:
This has a great quote: 

> Did you know the average American throws away 80lbs of clothing annually and that 85% of donations end up in landfill? 

That could be used as motivation in the editorial section. I love getting stats like this in because it makes people read on instead of just telling them to tidy their room and do their homework. :D